pommerman/network/ion_server/network.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+"""IonServer Network handler - This contains functions responsible Server-Client communication (You shouldn't use this file directly)"""
+
+# Imports
+from asyncio import new_event_loop, set_event_loop, get_event_loop
+from websockets import serve
+from threading import Thread
+from .constants import SPCommands as spc
+from .constants import NetworkCommands as nic
```

Comment:
Get rid of these abbreviations. It's confusing later.

Reply:
Done.